src/Command.ts

```diff
@@ -27,6 +27,10 @@ const executors = {
     },
     clear: (invocation: Invocation, args: Array<string>): void => {
         setTimeout(() => invocation.terminal.clearInvocations(), 0);
+    },
+    exit: (invocation: Invocation, args: Array<string>): void => {
+        var app = require('remote').require('app');
+        app.quit();
```

Comment:
It will works correctly now, but we plan to add tabs and split, which would make this behaviour incorrect. Please remove the current terminal instance from this array instead. Perhaps, we might want to exit, if it was the last one. https://github.com/black-screen/black-screen/blob/38a69e28ac1562ed23def97407a0fbce4d5f0619/src/Application.ts#L5